URLTester/Output/ProgressBar.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+
+
+
+
+namespace _URLTester.Output
+{
+    public interface IProgressBar
+    {
+        void UpdateProgressBar(int currentIndex, string currentItem);
+    }
+
+    public class ProgessBar : IProgressBar
+    {
+        const int totalBlocks = 30;
+        const int blockIncrement = 5;
+        const ConsoleColor blueConsole = ConsoleColor.Blue;
+        const ConsoleColor blackConsole = ConsoleColor.Black;
+
+        private readonly int _totalCount;
+
+        private int _currerntIndex;
+        private string _currentItem;
+
+        public ProgessBar(int totalCount)
+        {
+            _totalCount = totalCount;
+        }
+
+        public void UpdateProgressBar(int currentIndex, string currentItem)
+        {
+            _currerntIndex = currentIndex;
+            _currentItem = currentItem;
+
+
+            CreateBar();
```

Comment:
This is weird you're re-creating the bar every time its updated.  Lots of extra unneeded logic!